src/client/bridge-data.ts

```diff
@@ -108,14 +136,9 @@ export interface BridgeDataFieldGetters {
   ): Promise<AssetValue[]>;
 
   /*
-  @dev This function should be implemented for all async yield bridges.
-  @dev It should return the yield for a given interaction
+  @dev This function should be implemented for all async yield bridges
+  @param interactionNonce A globally unique identifier of a given DeFi interaction
+  @return Current yield of the given interaction
```

Comment:
current yield = accumulated yield?

Reply:
I am not sure how it was originally intended but I would say it's current APR.